src/Sylius/Bundle/UserBundle/Resources/config/serializer/Model.User.yml

```diff
@@ -20,7 +20,7 @@ Sylius\Component\User\Model\User:
     relations:
         - rel: user
```

Comment:
I think that proper fix will be to change this `rel` to customer, since we still do `id: expr(object.getCustomer().getId())`.